docs/Configurations.md

```diff
@@ -1,99 +1,86 @@
-
-    - Clone the mock firebolt OS repository` https://github.com/rdkcentral/mock-firebolt`
-    - Start the mock Firebolt OS based on steps mentioned in the repository.
-    - Start the Firebolt Certification App with `npm start`
-    - On launch add the `mf=true` queryParameter to the app launch url and relaunch the app.
-    The above steps will by default connect the Firebolt Certification App to the mock Firebolt OS running on localhost at port 9998
-    Note: To change the hosted location of the mock Firebolt OS use the guidelines as mentioned in `https://github.com/rdkcentral/mock-firebolt/blob/main/docs/UsageWithinApps.md#activating-mock-firebolt
-```
-
 ### Background
 
-TODO: add details
+Connecting to Mock Firebolt enables testing against a simulated Firebolt OS, providing a safe and flexible enviroment for development and testing.
 
 ### Setup
 
-TODO: add details
+- Clone the Repository [Mock-Firebolt](https://github.com/rdkcentral/mock-firebolt)
+- Follow the instructions provided in the repository to start the Mock Firebolt.
+- Launch the Firebolt Cretification App and add `mf=true` to the app launch URL to connect to the mock environment.
 
 ### Usage
 
-TODO: add details
+- Use Mock Firebolt for testing API integrations and app behavior in a simulated Firebolt environment.
+
 
 ## Supported URL parameters
- 
-EXISTING CONTENTS(needs revision):
-```
-- Platform: platform=`<platform>`
-  - The supported TARGET values are passed. While executing the test suite, if we provide wrong platform it will not execute the suite and will show "Unsupported target used." error
-- Lifecycle Validation: lifecycle_validation=true
-  - When we give lifecycle_validation=true it blocks the default execution of `lifecycle.ready` and `lifecycle.finished` method.
-  - This will help us to validate lifecycle API's as per our need
-- MFOS: mf=true
-  - When we are passing mf=true or with userId, FCA will connect to MFOS server and when we invoke any api in FCA it will return the response from MFOS.
-- System Ui: systemui=true
-  - If FCA systemui=true, FCA acts as the base app in case of ripple. The background color will be changed to purple, and it will display one more button as "Launch FCA app" to launch FCA as third-party app on Ripple devices.
-- TestContext: testContext=true
-  - If testContext=true, it will add the field context in mocha report generated
-```
+
 
 ### Background
 
-TODO: add details
+URL parameters allow customization of the FCA's behavior, enabling testers to tailor the testing environemt to specific needs.
 
 ### Setup
 
-TODO: add details
+- Use various parameters like 'platform', 'lifecycle_validation','mf','systemui' and more as per the requirment.
+- These parameters modify how the FCA interacts with the environemtn and the test suite.
```

Comment:
Not sure this is necessary, if we explain in Usage, but to me doesn't really read like a setup instruction.

Reply:
removed the second line